bucket/dotter.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.12.6",
+    "description": "Dotfile manager and templater",
+    "homepage": "https://github.com/SuperCuber/dotter",
+    "license": "Unlicense",
```

Comment:
```suggestion
    "license": "Unlicense",
    "notes": "Run 'dotter init' to initialize config file",
```

Reply:
I'm not sure if this line should be added. The `dotter init` command should only be run inside a particular directory (where you want to save/load your configs from). As mentioned in the wiki, this should be a local git repository.

![image](https://user-images.githubusercontent.com/46838874/133941737-b805b341-fb0c-4892-8541-a859a29da39e.png)